CI/appveyor.install.ps1

```diff
@@ -204,7 +204,8 @@ function InstallPython() {
 }
 
 function InstallOpenssl() {
-  DownloadFile "https://indy.fulgan.com/SSL/openssl-1.0.2o-i386-win32.zip" "openssl-1.0.2o-i386-win32.zip"
+  # Old source was getting intermittant: DownloadFile "https://indy.fulgan.com/SSL/openssl-1.0.2o-i386-win32.zip" "openssl-1.0.2o-i386-win32.zip"
```

Comment:
I'd remove that comment. If we want to check the history, we can use git for that and if you use a nice commit message, we have that info as well.